docs/authoring/servertaskauthoring.md

```diff
@@ -0,0 +1,77 @@
+    2.  TaskStarted - Raised when the external service starts processing
+    3.  TaskCompleted -Raised when the external system is done processing (indicates success/failure)
+    
+         *Example*:  In below example once server picks task for execution, if server not receives the &#39;TaskStarted&#39; event within 5 minutes, server cancels the task. If server not receives the &#39;TaskCompleted&#39; event within 7 minutes, server cancels the task.         
+   "Events": { "TaskStarted": {"Timeout": "00:05:00"}, "TaskCompleted": {"Timeout": "00:07:00"}}
+
+    **Execute:** Configures the HTTP payload and properties for the Execute event raised from VSTS to the cloud service.  'HttpRequest' handler supports following properties.
+    - EndpointId: - Specified endpointId details used while triggering below mentioned &#39;EndpointUrl&#39;. This input is optional
+    - Endpoint URL: - Http Url to trigger on task execution
+    - Method: - Http method like PUT, GET, POST, PATCH, DELETE, OPTIONS, HEAD, TRACE.
+    - Body: - Body of http request
+    - Headers: - headers of Http request. Header should be in JSON format.
+    - WaitForCompletion: -  you can specify true or false. Default value false. If you specify &#39;true&#39;, server waits for TaskCompleted event happen from external. If task event not comes within task timeout, task will be canceled. This input is optional
+    - Expression: - Expression Criteria which defines when to pass the task. No expression means response content does not influence the result. Example: - For response {&quot;status&quot;: &quot;successful&quot;}, the expression can be eq(root[&#39;status&#39;], &#39;successful&#39;). More Information[https://go.microsoft.com/fwlink/?linkid=842996](https://go.microsoft.com/fwlink/?linkid=842996)
+
+    **Cancel**: Configures the HTTP payload and properties for the Cancel event raised from VSTS to the external service. This property is optional.  You can configure these properties if you want call external service on user cancels the task. Properties are same as 'Execute' properties defined above.
+
+- **ServiceBus**
+This enables you to post a message to azure service bus queue. It takes 3 inputs.
+
+    **Events:** Same set of events mentioned above are supported in ServerBus also.
+    
+    **Execute:** Configures the servicebus message payload and properties for the Execute event raised from VSTS to the cloud service. ServiceBus handler has following properties.
+    - EndpointId: - EndpointId details used while publishing the message
+    - ConnectionString: - Azure service bus connection string
+    - ServiceBusQueueName: - Queue name to publish message
+    - MessageBody: - Message body. Message body should be JSON format
+    - MessageProperties: - message properties. like Content-Type:\application/json&quot;
+    - CertificateString: -  used to encrypt the message. This input is optional
```

Comment:
Change "used to encrypt the message" for certificatestring to "certificate that can be used to encrypt the message"